mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -855,6 +853,10 @@ export const mapStateToProps = (state, ownProps) => {
   };
 };
 
+const mapDispatchToProps = {
+  setExperimentTagApi
```

Comment:
nit: add trailing comma.
```suggestion
  setExperimentTagApi,
```

Reply:
We should add this rule to `.eslintrc.js` and raise lint error if it's violated.

https://eslint.org/docs/rules/comma-dangle

Reply:
I'll create a PR for this.